extensions/wikia/Email/templates/ForgotPassword_body.mustache

```diff
@@ -0,0 +1,11 @@
+{{> Email_header}}
```

Comment:
It might be easier to have one main layout template that has a header, footer, and `{{{content}}}` variable as opposed to including the header and footer partials in each template. This would reduce the duplicate code of always having to include the header and footer partials. This is how mercury has been structuring their templates and I think it's a good idea.